tests/end-to-end/backstop.js

```diff
@@ -48,6 +48,9 @@ const getPublishedPageUrl = ({ space, page }) => {
 
 const spaces = ['kyanite-visual-tests', 'kyanite-visual-tests-dark-mode'];
 
+const postHoverInterval = 1000;
+const postClickInterval = 1000;
```

Comment:
I think adding unit name to such names is a good practice - a little `Ms` or `Millis` postfix appended to the names, and all becomes very clear in every scope/place :)